instrumentation/spring/spring-boot-autoconfigure/src/main/java/io/opentelemetry/instrumentation/spring/autoconfigure/propagators/CompositeTextMapPropagatorFactory.java

```diff
@@ -75,6 +80,7 @@ static TextMapPropagator getCompositeTextMapPropagator(
           propagators.add(W3CBaggagePropagator.getInstance());
           break;
         default:
+          log.warn("Unsupported type of propagator: " + type);
```

Comment:
Nit: you can use slf4j's built-in message formatting facilities instead of gluing strings yourself:

```suggestion
          log.warn("Unsupported type of propagator: {}", type);
```